eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -280,3 +259,27 @@ stages:
                 targetType: filePath
                 filePath: "eng/tools/publish-to-npm.ps1"
                 arguments: '-pathToArtifacts $(Package.Archive) -accessLevel "public" -tag "dev" -registry "$(Registry)" -npmToken "$(NpmToken)"'
+
+    - job: PublishDocsToNightlyBranch
+      pool:
+        name: azsdk-pool-mms-ubuntu-1804-general
+        vmImage: MMSUbuntu18.04
+      steps: 
+        - checkout: self
+        - download: current
+        - pwsh: |
+            Get-ChildItem -Recurse $(Pipeline.Workspace)/${{parameters.ArtifactName}}/
+          displayName: Show visible artifacts
+
+        - template: /eng/common/pipelines/templates/steps/update-docsms-metadata.yml
+          parameters:
+            Artifacts: 
+              - ${{ each artifact in parameters.Artifacts }}:
```

Comment:
These changes might conflict with @praveenkuttappan's changes https://github.com/Azure/azure-sdk-for-js/pull/15512

Reply:
These changes don't conflict. Praveen's linked PR does not alter where `PackageInfo` files land or how they're named.